app/controllers/catalog_controller.rb

```diff
@@ -223,6 +223,7 @@ class CatalogController < ApplicationController
     config.add_index_field 'pub_created_display', label: 'Published/Created'
     config.add_index_field 'format', label: 'Format', helper_method: :format_icon
     config.add_index_field 'holdings_1display', show: false
+    config.add_index_field 'contained_in_s', show: false
```

Comment:
This field must be added to solrconfig.xml so that the app can fetch it.

```
<requestHandler name="search" class="solr.SearchHandler" default="true">
    <lst name="defaults">
      ...
      <str name="fl">
        id,
        score,
        author_display,
        marc_relator_display,
        format,
        pub_created_display,
        title_display,
        title_vern_display,
        isbn_s,
        oclc_s,
        lccn_s,
        holdings_1display,
        electronic_access_1display,
        cataloged_tdt,
        contained_in_s
      </str>
      ...
```